src/Map/src/UXMapBundle.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class UXMapBundle extends Bundle
```

Comment:
```suggestion
final class UXMapBundle extends AbstractBundle
```

That would remove the need for getPath() right ?

Reply:
Yes!
Opened #1997